support/ci/install-nightly-rustfmt.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script can be run standalone, but it can also be sourced from rustfmt.sh.
+# If it's sourced, then $rustup is already defined for us. If it's not, we need
+# to make sure we have it.
+if [ -z "${rustup:-}" ]; then
```

Comment:
I think this is a bit more idiomatic to check if a variable is set. [See docs](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html#Bash-Conditional-Expressions).
```suggestion
if [ -v rustup ]; then
```

Alternatively, there's an idiom for setting a value, only if it's not set:
```bash
: ${rustup:=$(command -v rustup)}
```
See, [my favorite piece of shell docs](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02).

Reply:
Ah, that's terrific, thank you.